.ci/before_deploy.sh

```diff
@@ -7,7 +7,8 @@ if [[ "$GITSECRET_DIST" == "rpm" ]]; then
   sudo apt-get install -y rpm;
 fi
 
-if [[ ! -z "$GITSECRET_DIST" ]] && [[ -z "$KITCHEN_REGEXP" ]]; then
+# if not null GITSECRET_DIST
+if [[ ! -z "$GITSECRET_DIST" ]]; then
```

Comment:
This issue here is that `$KITCHEN_REGEXP` is defined where $GITSECRET_DIST is defined in .travis.yml Since $GITSECRET_DIST is only defined in two places, once for deb and once for rpm in .travis.yml this looks okay to me.